remappings.txt

```diff
@@ -1,14 +1,15 @@
-test=test/
```

Comment:
This shouldn't be removed right? There are files in `test/` that need to be mapped here.

Reply:
Hmm  ok... yeah that's weird, I didn't mean to update the remappings. 
I even purposedly remove the changes of my local rempappings.txt before pushing haha.
Ok let me re-run the forge remappings to regenerate this file.

Reply:
oof... looks like I did accidentally pushed the changes to remappings.txt
But now question is... I think that's what `forge remappings` returns.  So... should we keep it as is as the `forge remappings`?  

Cuz from my understanding `remappings.txt` doesn't do anything, but it's just to tell IDE to read.  And the true remappings is processed within `forge remappings` 🤔

Reply:
Reverted it to be matched with /main's  as I don't expect this PR should update any remappings entry. 
Apprently the `forge remappings` generate some wrongful data her.e